hrrecruitment/tests/phpunit/CRM/HRRecruitment/BAO/HRVacancyTest.php

```diff
@@ -18,12 +18,6 @@ public function setUpHeadless() {
       ->apply();
   }
 
-  function setUp() {
-  }
-
-  function teardown() {
-  }
-
   protected static function _populateDB($perClass = FALSE, &$object = NULL) {
```

Comment:
I just removed this method and the tests still pass. So it's fine to remove it. If you don't mine, I can push this change to this branch.